docs/docs/tutorials/creating-new-environments.md

```diff
@@ -4,179 +4,157 @@ title: Creating a new environment on Nebari
+## Introduction
 
-In this tutorial, we will create a new environment on Nebari using `conda-store`, look into how we can
-install new conda packages, manage multiple environments, and also share the environment with other Nebari users.
+In this tutorial, you will learn how to create a new environment on Nebari using `conda-store`,
+install new `conda` packages in your environment, manage multiple environments, and also share the environment with other Nebari users.
 
-### Why have environments?
+### Why using`conda` environments
 
-Environments give developers and analysts a "sandbox" to play in. They create isolated spaces to separate
-dependencies on a per-project basis. In general, environments solve several common issues:
+Development environments give developers and Data Scientists a "sandbox" to work in, experiment with new packages, work across multiple projects, and collaborate with their peers.
+Environment and package managers such as `conda` enables you to create isolated spaces to separate dependencies on a per-project basis.
+They also add other benefits such as:
 
-- Onboard new developers/contributors
-- Packages update frequently
-- There are often incompatibilities between certain package versions
-- The larger the environment, the harder it is to share with others
-- Reproducibility
+- Reduce friction to install and upgrade dependencies without affecting other projects or users
+- Reduce the risk of dependency conflicts
+- Promote reproducibility and replicability through dependencies pinning
+- Reduce the "works on my machine" risk - thus working towards reproducible builds across multiple machines
 
 ### What is `conda`?
 
-[Conda](https://docs.conda.io/projects/conda) is and open source package management system that allows us to create
+[Conda](https://docs.conda.io/projects/conda) is and open source package management system that allows you to create
 environments and install packages into them. It allows the creators of a package to specify required dependencies
-which `conda` can then solve into an environment. `Conda` can run an update on the environment to pull all of the
+which `conda` can then solve into an environment. `Conda` can then run an update on the environment to pull all the
 packages up-to-date while still maintaining compatibility.
 
-While `conda` manages compatibility between the packages in the environment, we often face a different issue...
-
+While `conda` manages compatibility between the packages in the environment this does not, by default guarantee reproducible builds over time.
 An environment created with a list of packages _today_ can differ from that same environment created with the same
 list of packages _tomorrow_. This can happen because package dependencies have changed, new releases have occurred,
 or even because a package is no longer available.
 
 ### What is `conda-store`?
 
-As developers and analysts, we have all experienced the `it works on my machine ¯ \_(ツ)_/¯` conundrum.
```

Comment:
aw. I thought my guy was cute!😆

Reply:
it is but I moved the text to the section above - we can add the ascii art if you want